py/travis-ci/travis_tftp.py

```diff
@@ -0,0 +1,11 @@
+import os
```

Comment:
Eventually we should move this into some common location that all hosts can share, perhaps even into the U-Boot test/py source tree. However, putting it here is just fine for now.